www/src/components/email-capture-form.js

```diff
@@ -1,9 +1,182 @@
 import React from "react"
 import { rhythm, options } from "../utils/typography"
-import { colors } from "../utils/presets"
-import HubspotForm from "./hubspot-form"
+import presets, { colors } from "../utils/presets"
+import hex2rgba from "hex2rgba"
+import { formInput } from "../utils/form-styles"
+import { buttonStyles } from "../utils/styles"
+
+const Label = props => (
+  <label>
+    {props.children}
+    {props.isRequired && (
+      <span
+        css={{
+          color: `red`,
+          "&:before": {
+            content: `'*'`,
+            color: colors.warning,
+          },
+        }}
+      />
+    )}
+  </label>
+)
+
+const SingleLineInput = React.forwardRef((props, ref) => (
+  <input
+    ref={ref}
+    css={{
+      ...formInput,
+      width: `100% !important`,
+      ":focus": {
+        borderColor: colors.gatsby,
+        outline: 0,
+        boxShadow: `0 0 0 0.2rem ${hex2rgba(colors.lilac, 0.25)}`,
+      },
+    }}
+    {...props}
+  />
+))
+
+const ErrorMessage = ({ children }) => (
+  <div
+    css={{
+      // dunno where this is comming from - just straight copied li style from dev tools
+      marginBottom: `calc(1.05rem / 2)`,
+      color: colors.warning,
+      fontSize: rhythm(1 / 2),
+    }}
+  >
+    {children}
+  </div>
+)
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.onSubmit = this.onSubmit.bind(this)
+
+    // let's use uncontrolled components https://reactjs.org/docs/uncontrolled-components.html
+    this.email = React.createRef()
+  }
+
+  state = {
+    errorMessage: ``,
+    fieldErrors: {},
+  }
+
+  onSubmit(e) {
+    e.preventDefault()
+
+    // validation here or use Formik or something like that
+    // we can also rely on server side validation like I do right now
+
+    const { portalId, formId, sfdcCampaignId } = this.props
+    const url = `https://api.hsforms.com/submissions/v3/integration/submit/${portalId}/${formId}`
+    const data = {
+      fields: [
+        {
+          name: `email`,
+          value: this.email.current.value,
+        },
+      ],
+      context: {
+        pageUri: window.location.href,
+        pageName: document.title,
+      },
+    }
+
+    if (sfdcCampaignId) {
+      data.context.sfdcCampaignId = sfdcCampaignId
+    }
+
+    const xhr = new XMLHttpRequest()
```

Comment:
![image](https://user-images.githubusercontent.com/3924690/47126102-9d1bea80-d24c-11e8-96e5-e20dae0b9bff.png)

Reply:
:D give me some fetch love and tell world IE is not a thing anymore to not throw in polyfills :)